README.md

```diff
@@ -277,6 +278,20 @@ To use an obfuscated key in your code, create one and use the builder variables
 let key = ObfuscatedKey().T.h.i.s.underscore.I.s.dash.o.b.f.u.s.c.a.t.e.d.value
 ```
 
+### LanguageFilter
+
+To find if a string contains potentially objectionable input, create a filter and place a query.
+
+``` swift
+// TODO
```

Comment:
Looks like we're missing the basic how-to from here.

Reply:
Whoops!

I guess at least I put in the todo. I wrote that before I decided how I'd implement it. :)